src/handlers/glacier.rs

```diff
@@ -64,7 +64,7 @@ async fn handle_input(ctx: &Context, event: &Event, cmd: GlacierCommand) -> anyh
     };
 
     let url = cmd.source;
-    let response = ctx.github.raw().get(&format!("{}{}", url.replace("github", "githubusercontent"), "/playground.rs")).send().await?;
+    let response = ctx.github.raw().get(&format!("{}{}", url, "/raw")).send().await?;
```

Comment:
Maybe it makes sense to just directly send a request to the relevant URL? That would give more flexibility and be easier to understand I think-- just saying "please provide a URL resolving to plain-text rust code" makes a lot of sense I think.

Reply:
Good idea, I'll change it!